third-party/mockets/server_socket.py

```diff
@@ -0,0 +1,51 @@
+import socket
+import time
+
+HOST = "10.0.2.1"  # The server's hostname or IP address
+PORT = 65432  # The port used by the server
+
+BUFFER_SIZE = 1024
+
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+s.bind((HOST, PORT))
+s.listen()
+
+
+times = []
+best = 100
+worst = 0
+
+for x in range(1, 20):
```

Comment:
You could add a few white spaces in this for loop, otherwise it is going to be a little difficult to read